frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsList.component.tsx

```diff
@@ -18,22 +18,60 @@ import { TicketsCardHooksSelectors } from '@/v5/services/selectorsHooks';
 import { EmptyListMessage } from '@controls/dashedContainer/emptyListMessage/emptyListMessage.styles';
 import { FormattedMessage } from 'react-intl';
 import { TicketItem } from './ticketItem/ticketItem.component';
-import { List } from './ticketsList.styles';
+import { List, ListContainer, Loader } from './ticketsList.styles';
+import { TableVirtuoso } from 'react-virtuoso';
+import { useEffect, useRef } from 'react';
 
 export const TicketsList = () => {
 	const filteredTickets = TicketsCardHooksSelectors.selectFilteredTickets();
+	const selectedTicketId = TicketsCardHooksSelectors.selectSelectedTicketId();
+	const selectedIndex = filteredTickets.findIndex((ticket) => ticket._id === selectedTicketId);
+	const shouldShowLoader = filteredTickets.length >= 10;
+	const virtuosoRef = useRef<any>();
+	const isFiltering = TicketsCardHooksSelectors.selectIsFiltering();
+	
+	useEffect(() => {
+		virtuosoRef.current?.scrollToIndex?.({
+			behavior: 'instant',
+			block: 'nearest',
+			align: 'start',
+			index: selectedIndex === -1 ? 0 : selectedIndex,
+		});
+	}, [selectedIndex, filteredTickets, isFiltering]);
+
+	if (isFiltering) {
+		return (
+			<EmptyListMessage>
+				<FormattedMessage id="viewer.cards.tickets.searching" defaultMessage="Searching..." />
+			</EmptyListMessage>
+		);
+	}
+
+
+	if (!filteredTickets.length) {
+		return (
+			<EmptyListMessage>
+				<FormattedMessage id="viewer.cards.tickets.noResults" defaultMessage="No tickets found. Please try another search." />
+			</EmptyListMessage>
+		);
+	}
 
 	return (
-		<>
-			{filteredTickets.length ? (
-				<List>
-					{filteredTickets.map((ticket) => <TicketItem ticket={ticket} key={ticket._id} />)}
-				</List>
-			) : (
-				<EmptyListMessage>
-					<FormattedMessage id="viewer.cards.tickets.noResults" defaultMessage="No tickets found. Please try another search." />
-				</EmptyListMessage>
-			)}
-		</>
+		<ListContainer >
+			{shouldShowLoader && <Loader />}
+			<TableVirtuoso
```

Comment:
There is a white background that can be seen at the corners of the tickets list.
![image](https://github.com/user-attachments/assets/22adc2a8-d7bc-4b8d-a1e0-64271fe6b6a8)
From this:
![image](https://github.com/user-attachments/assets/4851f4b2-1a2d-4932-9c79-fe554b867541)